tools/depends/target/python3/darwin_embedded.patch

```diff
@@ -214,7 +215,7 @@
          wchar_t *path = Py_DecodeLocale(execpath, &len);
 --- a/Modules/posixmodule.c
 +++ b/Modules/posixmodule.c
-@@ -197,6 +197,20 @@
+@@ -194,6 +194,20 @@
```

Comment:
Is the current range information incorrect?

Reply:
Yes, all ranges are correct. They are based off the preceding patches having been applied (5 patches) as laid out in the makefile.